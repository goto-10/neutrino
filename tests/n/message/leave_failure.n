# Copyright 2014 the Neutrino authors (see AUTHORS).
# Licensed under the Apache License, Version 2.0 (see LICENSE).


def $capture($a, $b, $c) as $args => $args;
def $level2() => module.leave($capture(1, 2, 3));
def $level1() => $level2();

do $level1();
