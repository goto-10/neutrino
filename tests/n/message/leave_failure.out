Error: %<condition: UncaughtSignal(escape)>
Info: --- backtrace ---
- #<module $:neutrino/tests/n/message/leave_failure.n>.leave(#<reified_arguments :capture(1, 2, 3)>)
- :level2()
- :level1()
